src/example_package/csv_reader/config.py

```diff
@@ -0,0 +1,81 @@
+constraints = {
```

Comment:
Nie zamierzamy modyfikować tego słownika w czasie działania aplikacji, prawda? Jeśli tak, to można to zaznaczyć
```suggestion
from typing import Final

CONSTRAINTS: Final[dict] = {
```

Reply:
Ok, cofam to. Bez typingu, niech na razie będzie dynamicznie. Wystarczy napisać capslockiem.